health/notifications/alarm-notify.sh.in

```diff
@@ -444,7 +443,6 @@ filter_recipient_by_criticality() {
 			# remove the file, so that he will only receive notifications for CRITICAL states for this alarm
 			rm "${NETDATA_CACHE_DIR}/alarm-notify/${method}/${r}/${alarm_id}"
 			debug "SEVERITY FILTERING for ${x} VIA ${method}: ALLOW: recipient has been notified for this alarm (will only receive CRITICAL notifications from now on)"
-			return 0
```

Comment:
In both cases you allow control to go through and return 1. In case of  debug, it will write two conflicting 
debug messages one saying ALLOW and the other BLOCK. Can you check this?

Reply:
Are we changing this or consider it minor?

Reply:
I think so @stelfrag .

Reply:
Ok, I deserved that -- 2 questions in one sentence :laughing: 

1. Change it?
2. Not change it because it's minor and we should just merge?

Please select 1 or 2 :smile:

Reply:
2 :D